src/index.ts

```diff
@@ -5,9 +5,18 @@ import {
   useSlashAuth,
 } from './core/context/legacy-slashauth';
 import { supportedFontFamilies } from './core/ui/fonts';
+import { AccountSettings } from './core/ui/components/account-settings';
+import { DropDown } from './core/ui/components/drop-down';
```

Comment:
This should not be exported.

If you are testing this way then we're not testing the mounted components which could end up hurting us.

Instead please expose a `mountDropdownComponent` function on the slashauth class which mounts this.

Reply:
updated